client/src/components/Header.tsx

```diff
@@ -52,6 +57,25 @@ const useStyles = makeStyles((theme: Theme) => ({
     textDecoration: 'none',
     color: 'black',
   },
+  infoButtons: {
+    color: '#323232!important',
+  },
+  signUp: {
+    backgroundColor: '#EF6A60!important',
+    color: 'white!important',
+    borderRadius: '10px',
+    border: '1px solid #EF6A60!important',
+    marginRight: '10px!important',
+    width: '100px',
+  },
+  signIn: {
+    backgroundColor: 'white!important',
+    color: '#323232!important',
+    borderRadius: '10px',
+    border: '1px solid #323232!important',
+    marginLeft: '5px!important',
+    width: '100px',
+  },
```

Comment:
@paaaarv , is there a way to use MUI props to achieve this style, similar to this: https://github.com/Nonprofit-Exchange-Hub/web-app/commit/4f732a051319f326ae696f47afae383e3a18815f#diff-b36271bb100b70cfb099cab9923f9dba10e0476f6f67d821a09cc879afa185ff 

If not, then this should be fine.

Reply:
hey! do you mean passing in any styles to sx props?